power_grid_model_c/power_grid_model/include/power_grid_model/three_phase_tensor.hpp

```diff
@@ -198,8 +199,9 @@ template <column_vector DerivedA> inline double max_val(Eigen::ArrayBase<Derived
 
 // function to sum rows of tensor
 template <rk2_tensor DerivedA> inline auto sum_row(Eigen::ArrayBase<DerivedA> const& m) { return m.rowwise().sum(); }
-// overload for double
+// overload for double and double complex
 inline double sum_row(double d) { return d; }
+inline DoubleComplex sum_row(DoubleComplex d) { return d; }
```

Comment:
the docstring only points at the first function after the comment. either have a comment on the newline, or none at all (actually, the comment in the tensor implementation is already redundant), or (inferior) make it into something like

```suggestion
template <typename T>
    requires std::floating_point<T> || std::same_as<T, DoubleComplex>
inline T sum_row(T d) { return d; }
```